tests/metagpt/provider/req_resp_const.py

```diff
@@ -155,7 +160,8 @@ def get_anthropic_response(name: str, stream: bool = False) -> Message:
             ),
             ContentBlockDeltaEvent(
                 index=0,
-                delta=TextDelta(text=resp_cont_tmpl.format(name=name), type="text_delta"),
+                delta=TextDelta(text=resp_cont_tmpl.format(
```

Comment:
pre-commit you can set max_length of line to 120. 80 is too short. There is no need to auto change these code.